src/Swarm/TUI/Controller.hs

```diff
@@ -419,7 +419,7 @@ handleModalEvent = \case
                 uiState . uiGoal . listWidget .= newList
               GoalSummary -> handleInfoPanelEvent modalScroll (VtyEvent ev)
             _ -> handleInfoPanelEvent modalScroll (VtyEvent ev)
-      _ -> return ()
+      _ -> handleInfoPanelEvent modalScroll (VtyEvent ev)
```

Comment:
I almost wish we returned `EventM Name AppState Bool` to indicate whether the event was handled or fell through.

Then the top handler would know to call the next handler that might be interested.